icu4c/source/config/dist.mk

```diff
@@ -25,15 +25,16 @@ DISTY_IN=$(DISTY_DATA)/in
 DOCZIP=icu-docs.zip
 
 SVNTOP=$(top_srcdir)/..
-SVNVER=$(shell svnversion $(SVNTOP) | cut -d: -f1 | tr -cd 'a-zA-Z0-9')
-SVNURL=$(shell svn info $(SVNTOP) | grep '^URL:' | cut -d: -f2-)
+GITVER=$(shell (cd $(SVNTOP) && git describe --tags --exact-match || git rev-parse --short HEAD) 2>/dev/null || echo 'unknown')
```

Comment:
I think the `2>/dev/null` ought to be moved inside of the parenthesis to `git describe`, for that's the only command in this command line that we expect to be able to fail (and for which we therefore want to throw the error message away). Any other error message is unexpected and would therefore be valuable to show.